packages/smooth_app/lib/pages/product/product_page/footer/new_product_footer_settings.dart

```diff
@@ -60,12 +64,17 @@ class ProductFooterSettingsButton extends StatelessWidget {
   /// Header + list padding + for each action: height + separator
   /// + bottom padding (nav bar)
   double _computeContentSize(BuildContext context) {
-    return SmoothModalSheetHeader.MIN_HEIGHT +
+    final double availableHeight =
```

Comment:
Please keep your comments to the very very very minimum, in general for this PR.